code/js/rcalc-language.js

```diff
@@ -0,0 +1 @@
+﻿ // change language
```

Comment:
Adding of this file should come later. :-)

Reply:
deleted ;-)